test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/NuGet.XPlat.FuncTest.csproj

```diff
@@ -28,11 +28,8 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
   </ItemGroup>
-  <ItemGroup Condition=" '$(IsCore)' == 'true' ">
-    <PackageReference Include="Microsoft.Net.Compilers.netcore" />
```

Comment:
Every single time I tried to improve NuGet.XPlat.FuncTest, I ran into endless problems with all this compiler package and Microsoft.CSharp.Core.targets file stuff.  Good work finding something that worked!👏

I still hope one day we can figure out why it's needed, and find a way to make tests not need it, but one step at a time.

Reply:
I'm really confused why `Microsoft.CSharp.core.targets` is needed at all.  What are the tests doing that actually import that file? 🤷‍♂️    anyway, way off topic for this PR. I'm just happy that it didnt' block you like it's blocked me so many times before.